modules/protect/math-fallback.php

```diff
@@ -124,8 +124,15 @@ static function math_form() {
 			?>
 			<div style="margin: 5px 0 20px;">
 				<strong><?php esc_html_e( 'Prove your humanity:', 'jetpack' ); ?> </strong>
-				<?php echo $num1 ?> &nbsp; + &nbsp; <?php echo $num2 ?> &nbsp; = &nbsp;
-				<input type="input" name="jetpack_protect_num" value="" size="2" />
+				<label for="sum">
+					<?php echo $num1; ?>
+					<span class="accessibility-visually-hidden">__(plus)</span>
+					<span aria-hidden="true">&nbsp; + &nbsp;</span>
+					<?php echo $num2; ?>
+					<span class="accessibility-visually-hidden">__(equals)</span>
```

Comment:
Same here, this is won't render "equals" as text but "__(equals)". Fix it following the code in the other comment.